src/main.rs

```diff
@@ -0,0 +1,86 @@
+//! STAR Randomness web service
+
+use axum::{routing::get, routing::post, Router};
+use clap::Parser;
+use std::sync::{Arc, RwLock};
+use tracing::{debug, info};
+
+mod handler;
+mod update;
+
+#[cfg(test)]
+mod tests;
+
+/// Maximum number of points acceptable in a single request
+const MAX_POINTS: usize = 1024;
+
+/// Command line switches
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+pub struct Config {
+    /// Duration of each randomness epoch
+    #[arg(long, default_value_t = 5)]
```

Comment:
we should probably enable the `env` feature for the clap crate, and add the `env` attribute to each `arg` macro (i.e. `#[arg(long, default_value_t = 5, env)]`).

This will allow clap to also parse env vars for configuration, which will be extremely useful when deploying via k8s (env vars are prob the most common way we provide config to containers, in general).

Reply:
Would this be helpful @NullHypothesis?

I don't mind adding this if it's helpful for testing, but we're planning to deploy this in an enclave, where I think everything has to be hard-coded in the image, at least until we implement key sharing. So it doesn't much matter if it's set as an env variable or a command line option; both have to go in start.sh.

Reply:
Yes, we won't be able to use environment variables to configure our production service. (We could however teach the nitro-shim to read these variables and send them to the enclave somehow.) I'll leave the testing aspects to you too.

Reply:
ah ok, i forgot that enclaves are deployed differently, thanks for clarifying. i'm okay with leaving this out if it won't be useful.